rust_snuba/rust_arroyo/src/backends/kafka/mod.rs

```diff
@@ -328,9 +335,11 @@ impl<C: AssignmentCallbacks> KafkaConsumer<C> {
 
         let topic_str: Vec<&str> = topics.iter().map(|t| t.as_str()).collect();
         consumer.subscribe(&topic_str)?;
+        let topics = topics.to_owned();
```

Comment:
Normally I'd advocate for passing the already-owned value in, but it'd probably cause a bunch of changes for not a lot of benefit.